src/main/webapp/app/shared/model/user.model.ts

```diff
@@ -11,7 +11,7 @@ export interface IUser {
   createdDate?: Date | null;
   lastModifiedBy?: string;
   lastModifiedDate?: Date | null;
-  password?: string;
+  imageUrl?: string;
```

Comment:
Seems unusual change

Reply:
I manually updated the JHipster model on the frontend here to match the backend class.

The models in `shared/model` are generated by JHipster, which are separate from models generated by openapi.